stratum/hal/lib/barefoot/bf_sde_wrapper.cc

```diff
@@ -44,6 +44,10 @@ int switch_pci_sysfs_str_get(char* name, size_t name_size);
 
 DEFINE_string(bfrt_sde_config_dir, "/var/run/stratum/bfrt_config",
               "The dir used by the SDE to load the device configuration.");
+DEFINE_bool(incompatible_bfrt_legacy_bytestring_responses, false,
```

Comment:
```suggestion
DEFINE_bool(incompatible_bfrt_legacy_bytestring_responses, true,
```

So we can merge this now without changing the default behavior.